specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
I think there may be two different definitions of the word `audit` here. There is `audit` in the sense of "the SOS selected this contest to audit", and then `audit` in the sense of "this ballot is one of the ones selected to be audited as part of the random sequence". So when the county pulls a ballot they enter their interpretations for all the contests (thus auditing the entire ballot), but to perform the RLA calculations for a specific contest, don't we select the random sequence based on the set of ballots on which that contest appears rather than the set of all ballots?